.github/workflows/docker.yaml

```diff
@@ -0,0 +1,71 @@
+name: Docker Build HTML Image
+
+on:
+  # Allow manual trigger
+  workflow_dispatch:
+    inputs:
+      # Ask if docker depoyment needed when manually triggered
+      deploy:      
+        description: 'Deploy Docker Image (y/n)?'
+        required: true
+        default: 'n'
+  push:
+    tags:
+      - '[0-9].[0-9]+.[0-9]+' 
+      
+env:
+  DOCKER_REGISTRY_PATH: useblocks
+  IMAGE_NAME: sphinxneeds
+  BASE_IMAGE: sphinxdoc/sphinx:latest
+  NEEDS_VERSION: 0.7.8
```

Comment:
@danwos Is the current release version available during the pipeline run, to avoid hard-coding here? This is needed to version to docker image

Reply:
Not sure, maybe a task in the workflow is needed like `pip install .` to install it and make it available.

Reply:
ah sorry. I just need the version number of the release version, not the tool. I do not want to hardcode the version number and forget it update it during release. So a variable or a config file where the current version is available so that I can reference it

Reply:
Got it.
There are files containing this number, but I'm not sure if there is a way to get it easily.
The best file would be the pyproject.toml : https://github.com/useblocks/sphinxcontrib-needs/blob/master/pyproject.toml

But for me, it would be ok to raise it by hand with each release.
We already need to do it for pyproject.toml, needs.py, and conf.py.